Remotion/ObjectBinding/Web/UI/Controls/BocReferenceValueImplementation/IBocAutoCompleteReferenceValue.cs

```diff
@@ -36,6 +36,8 @@ public interface IBocAutoCompleteReferenceValue : IBocReferenceValueBase
     string? ValidSearchStringRegex { get; }
     string? ValidSearchStringForDropDownRegex { get; }
     string? SearchStringForDropDownDoesNotMatchRegexMessage { get; }
+    string? LoadingMoreMatchesMessage { get; }
+    string? LoadedMoreMatchesMessage { get; }
```

Comment:
revert please. we don't need the option to configure the messages. the renderer loads them forn the resources, that's sufficient.
```suggestion
```